koji/collection-mash-split.py

```diff
@@ -470,6 +470,14 @@ def main():
                 el8_candlepin.name = 'katello-candlepin-{}-el8'.format(version)
                 mashes.append(el8_candlepin)
 
+    elif collection == 'pulpcore':
+        CONFIG["gitloc"] = "https://github.com/theforeman/pulpcore-packaging.git"
+        git_tag = "rpm/{}".format(version)
```

Comment:
This suggests there may be deb packaging in the future. Or is this only to mirror foreman-packaging?

Reply:
To mirror f-packaging. The alternative would be to call the branches `PULPCORE-X.Y` or something. I don't mind either way.

Reply:
Mirroring has the benefit that we know the Jenkins jobs can parse that ;)

Reply:
I'm ok with that. Do you intend to have a master branch similar to foreman-packaging which just serves a README about the layout (and is the default branch) or will you keep updating the default branch over time to the latest version?

Reply:
Good question I didn't think about yet.

Keeping the latest branch as default sounds good.